app/controllers/grade_entry_forms_controller.rb

```diff
@@ -102,6 +102,15 @@ def update_grade
   # For students
   def student_interface
     @grade_entry_form = GradeEntryForm.find(params[:id])
+    if @grade_entry_form.is_hidden
+      render 'shared/http_status', formats: [:html],
+             locals: {
+                 code: '404',
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.